src/encoded/commands/deploy.py

```diff
@@ -179,7 +183,7 @@ def _tag_ec2_instance(
         tags_dict['elastic_ip_swtich_datatime'] = 'na'
         if cluster_master:
             tags_dict['elasticsearch_head'] += 'yes'
-            tags_dict['Role'] += '-datahead'
+            tags_dict['Role'] += 'head'
```

Comment:
is this supposed to be -head?  I guess it doesn't matter that much.

Reply:
no, the -data already exists.  Previously the role came out as -datadatahead